community/dbms/src/main/java/org/neo4j/commandline/dbms/ImportCommand.java

```diff
@@ -101,7 +101,9 @@ public String usage()
             .withArgument( new OptionalBooleanArg( "ignore-missing-nodes", false,
                     "If relationships referring to missing nodes should be ignored during the import." ) )
             .withArgument( new OptionalBooleanArg( "ignore-empty-arrays", Configuration.COMMAS.emptyArraysAsNull(),
-                    "If empty array properties should be ignored, or if 'false', stores as empty arrays." ) );
+                    "If empty array properties should be ignored, or if 'false', stores as empty arrays." ) )
```

Comment:
We last decided to remove this functionality altogether. Before we merge this PR I'd like to go over the decision. And see whether the reasoning still stands true.